apps/sms/services/js/messaging/messaging_client.js

```diff
@@ -0,0 +1,54 @@
+/* global bridge */
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for messaging functionality.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'messaging-service';
+  const TIMEOUT = 5000;
```

Comment:
Now we can set the timeout by ourself instead of default 1 sec since it's not enough for retrieving the mms, so I ask Wilson to have additional parameter for timeout. Maybe the best way for messaging service is to set no timeout since it's highly depends on network status and gecko api also have it's timout mechanism.